StripePaymentSheet/StripePaymentSheet/Source/Analytics/STPAnalyticsClient+CustomerSheet.swift

```diff
@@ -15,15 +15,25 @@ extension STPAnalyticsClient {
     }
 
     // PM selection & Confirmation
-    func logCSSelectPaymentMethodScreenConfirmedSavedPMSuccess(type: String?) {
+    func logCSSelectPaymentMethodScreenConfirmedSavedPMSuccess(type: String?, syncDefaultEnabled: Bool? = nil) {
         let paymentMethodType = type ?? "unknown"
+        var params: [String: Any] = [:]
+        params["payment_method_type"] = paymentMethodType
```

Comment:
Nit: Can put this in one line `var params: [String: Any] = ["payment_method_type": paymentMethodType]`, same with below.